torch/autograd/gradcheck.py

```diff
@@ -2130,7 +2136,9 @@ def gradgradcheck(
     fast_mode: bool = False,
     masked: bool = False,
 ) -> bool:
-    r"""Check gradients of gradients computed via small finite differences
+    r"""Verify gradient of gradients.
```

Comment:
Some here as well.

Reply:
I have changed this to original. Please review it.